src/sass/components/_section--features.scss

```diff
@@ -131,21 +133,34 @@
     }
     .furniture {
       max-height: 490px;
-      background: url('../images/features/sofa.jpg');
-      background-size: cover;
-      background-position: center;
+      height: 490px;
       padding: 0;
-      &:hover {
-        .price {
-          .btn-main-small {
+      &:hover .price .btn-main-small {
             background-color: $primary;
-          }
-        }  
+      }  
+      .fur1 {
+        background: url('../images/features/sofa.jpg');
+      }
+      .fur2 {
+        background: url('../images/features/outdoor.jpeg');
+      }
+      .fur3 {
+        background: url('../images/features/decorations.jpeg');
       }
+      .carousel-inner {
+        height: 100%;
+        .carousel-item {
+          height: 100%;
+          margin-top: 0;
+          background-size: cover;
+          background-position: center;
+        }
+      }
+      .contentBox {
+        padding-top: 145px;
```

Comment:
poniżej masz chyba źle zrobione wcięcie - tak to wygląda

Reply:
Poprawiono.